.github/workflows/ci-build-binaries.yml

```diff
@@ -49,3 +49,15 @@ jobs:
 
     - name: Build binaries
       run: make ${{ matrix.platform.task }}
+
+    - name: Collect checksums
+      run: make collect-sums
+
+    - name: Clean
+      run: make clean
+
+    - name: Build binaries
+      run: make ${{ matrix.platform.task }}
```

Comment:
I wouldn't want to do this, this is already a very long CI job, it's not worth making it twice as long just to verify repro.

Reply:
Actually less because I'm not cleaning the go build cache, which is something I should check on as well. So it helped with something! But I agree the operational cost is something to consider. I do think it makes sense to ship checksums with releases and include it in the release tagging process.

Reply:
Yes, adding checksums to the release process would be useful.

Reply:
Do you prefer additional commits or squashed commits?

Reply:
doesn't matter, we squash on merge, just make sure all commits are signed.